deploy/crds/operators.coreos.com_servicebindings_crd.yaml

```diff
@@ -123,12 +123,21 @@ spec:
               - version
               type: object
             bindAsFiles:
-              description: BindAsFiles flag enable binding values as files.
+              description: BindAsFiles makes available the binding values as files
+                in the application's container See MountPath attribute description
+                for more details.
               type: boolean
-            customEnvVar:
-              description: Custom env variables
+            detectBindingResources:
+              description: DetectBindingResources is flag used to bind all non-bindable
+                variables from different subresources owned by backing operator CR.
+              type: boolean
+            namePrefix:
+              description: NamePrefix is the prefix for environment variables
+              type: string
+            mappings:
+              description: Custom mappings
               items:
-                description: EnvVar represents an environment variable present in
+                description: Mappings represents an environment variable or file name present in
```

Comment:
CRD file need to regenerated - it does not match the latest state in `servicebinding_types.go`